grype/pkg/qualifier/rpmmodularity/qualifier.go

```diff
@@ -0,0 +1,47 @@
+package rpmmodularity
+
+import (
+	"strings"
+
+	"github.com/anchore/grype/grype/pkg"
+	"github.com/anchore/grype/grype/pkg/qualifier"
+)
+
+type rpmModularity struct {
+	module string
+}
+
+func NewRpmModularityQualifier(module string) qualifier.Qualifier {
```

Comment:
nit: from a naming/organization point of view, this should be either:
- renamed to `New()` so that callers use `rpmmodularity.New()`, or
- be moved to the `qualifier` package, so that callers call `qualifier.NewRpmModularityQualifier()`

I can see the value of keeping the underlying interface separate from the capabilities, so the first option seems like the right call at a surface glance.